package.json

```diff
@@ -35,13 +35,7 @@
     "build": "react-app-rewired build",
     "test": "CI=true yarn run test:jest",
     "test:jest": "react-app-rewired test --env=jsdom",
-    "eject": "react-scripts eject",
-    "lint:less": "stylelint 'src/**/*.less' --config .stylelintrc",
-    "lint:eslint": "eslint --max-warnings 0 'src/**/*.js'",
-    "lint:eslintfix": "eslint --max-warnings 0 --fix 'src/**/*.js'",
-    "lint": "yarn run lint:eslint && yarn run lint:less",
-    "lint:fix": "yarn run lint:eslintfix && yarn run lint:less",
-    "lint:security": "nsp check"
+    "eject": "react-scripts eject"
```

Comment:
I don't understand this change, why remove everything related to linting? Also, do we need the `eject` script? I believe you've ejected the project already?

Reply:
we don't need the eject script but that is likely the source of the merge conflict.

Reply:
> I don't understand this change, why remove everything related to linting?

Those linting commands were inaccurate since they didn't always target all the files we care about and also it means we have to maintain commands in two places `package.json` and `.therapist.yml`. I think making Therapist the default tool for linting is the correct call in this case.

Reply:
Are you saying that there's now a script in package.json that allows to run the linting? And to run it on all the files, even on those that are not staged yet?

Reply:
I should add some docs but:
```sh
# Lint all files (incuding untracked files)
therapist run --use-tracked-files --include-untracked

# Fix all files (including untracked files)
therapist run --fix --use-tracked-files --include-untracked --stage-modified-files
```

I recognize these are verbose but I am in the middle of a patch to add shortcuts to therapist that will make these very quick and easy, along the lines of:
```py
# Lint all files
therapist do lint

# Fix all files
therapist do fix
```

Reply:
ok cool, could we have an entry in the package.json scripts to do that? I believe it's the entry points for most contributors?

Reply:
we could